Architecture/Logical-Authorisation-Model.md

```diff
@@ -39,3 +44,61 @@ Table describing the entities in the logical data-model
 | organization | A party playing a role in the management and control of the devices, for example municipalities. |
 | owner | An organization role. Each device has an owner. |
 | event | An event reported by the device (which is not the action of a natural person) for example an error or security problem. |
+
+An organization can get rights from the device owner to specific functions of a device, for instance reading value _x_ on device _y_. Every other function of the device has to offer will not be available to this organization.
+
+To ensure that devices can only receive instructions from a 'genuine' OSGP it must be possible to authenticate the OSGP platform. This is implemented through a standard technology based on asymmetric encryption. The OSGP platform will receive an unique key to enable the devices to tell if the messages come from a 'genuine' OSGP. To prevent replay-attacks each message will get an index number (this is standard practice as well).
```

Comment:
Please use open smart grid platform instead of OSGP to prevent confusion with the OSGP Alliance.